recipes/libsbml/bld.bat

```diff
@@ -0,0 +1,19 @@
+echo on
+
+cmake -G"Ninja" -S . -B build ^
+    -DCMAKE_INSTALL_PREFIX="%PREFIX%" ^
+    -DCMAKE_INSTALL_LIBDIR="%PREFIX%"/lib ^
+    -DCMAKE_CXX_COMPILER="%CXX%" ^
+    -DCMAKE_C_COMPILER="%CC%" ^
```

Comment:
```suggestion
    %CMAKE_ARGS% ^
```
I'm pretty sure our CMAKE_ARGS variable sets these types of things for you. CMAKE_ARGS is always a good idea on our channel because it helps set a lot of things which support our cross-compiling tool chain and weird conda directories.

https://conda-forge.org/docs/maintainer/knowledge_base/#using-cmake